### PR TITLE
WIP: subscriptions: Migrate to webpack

### PR DESCRIPTION
**Pull request by stefwalter**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+module.exports = {
+    resolve: {
+        alias: {
+<<<<<<< HEAD
+            "mustache": "mustache/mustache.js",
+=======
+            mustache: "mustache/mustache.js",
+            react: "react-lite-cockpit/dist/react-lite.js",
+>>>>>>> d16f903... subscriptions: Migrate to webpack
```
Comment (dperpeet) [inline webpack.config.js]: is this leftover from a diff/merge?
also subscriptions don't use react currently

**Comment by dperpeet** [comment]
note that subscriptions aren't covered by our integration tests yet, that's a work in progress right now

**Comment by dperpeet** [comment]
> note that subscriptions aren't covered by our integration tests yet, that's a work in progress right now

we should wait for #4697 to go in first so we have the test coverage

**Comment by larsu** [comment]
Obsolted by #4730. Please feel free to reopen if I forgot something.

**Pull request closed without merging**